### ISSUE TITLE
Stops playing after a while in background (android)

### ISSUE THREAD

**Issue by sweapp** [zmxv/react-native-sound#782]
It works nice but when streaming in background it stops after some minutes.
Appreciate any help I can get

**Comment by altiparmakzeynep** [comment]
@sweapp Did you fix it?

**Comment by sweapp** [comment]
> @sweapp Did you fix it?

No, after searching for a solution in weeks without any result, I wrote an native android app instead and it's working very nice.

**Comment by 13627491210** [comment]
> 

i have the same question , can you help me? thanks

**Comment by altiparmakzeynep** [comment]
@13627491210 this is still can't work. You can use another library : react-native-track-player
